marketplace/deployer_util/expand_config_test.py

```diff
@@ -236,7 +240,7 @@ def test_generate_certificate(self):
     self.assertEqual(cert.get_subject(), cert.get_issuer())
     self.assertEqual(cert.get_subject().OU, 'GCP Marketplace K8s App Tools')
     self.assertEqual(cert.get_subject().CN, 'Temporary Certificate')
-    self.assertEqual(cert.get_signature_algorithm(), 'sha256WithRSAEncryption')
+    self.assertEqual(cert.get_signature_algorithm(), b'sha256WithRSAEncryption')
```

Comment:
This should be valid. For reference: https://stackoverflow.com/questions/6269765/what-does-the-b-character-do-in-front-of-a-string-literal

Reply:
Py3 treats str and bytes differently:

b'test' != 'test'